.vscode/settings.json

```diff
@@ -83,5 +83,8 @@
     "**/.DS_Store": true,
     "**/Thumbs.db": true,
     "**/node_modules": false
+  },
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
What's the purpose of this change. any docs ?

Reply:
Seems a unexpected change by vscode itself. I will revert it.